playbooks/common/openshift-node/config.yml

```diff
@@ -79,7 +78,7 @@
   hosts: oo_nodes_to_config
   gather_facts: no
   vars:
-    sync_tmpdir: "{{ hostvars.localhost.mktemp.stdout }}"
+    sync_tmpdir: "{{ hostvars.localhost.mktemp.stdout }}/{{ groups['oo_first_master'][0] }}/{{ hostvars.localhost.mktemp.stdout }}"
```

Comment:
It seems odd that you'd use hostvars.localhost.mktemp.stdout twice on this line. Is this intentional?